framework/src/materials/ParsedMaterialHelper.C

```diff
@@ -169,6 +177,27 @@ ParsedMaterialHelper<is_ad>::functionParse(
     _symbol_names.push_back(pp);
   }
 
+  // get all extra symbols
```

Comment:
```suggestion
  // get the extra symbols from the parameters
```

Reply:
all of those are from parameters

Reply:
I m not contradicting that

Reply:
I know, I'm just politely rejecting this suggestion